charts/portal/templates/cronjob-backend-processes.yaml

```
@@ -157,29 +157,50 @@ spec:
+              value: "{{ .Values.backend.processesworker.clearinghouse.defaultClearinghouse.clientId }}"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__DEFAULTCLEARINGHOUSECREDENTIALS__CLIENTSECRET"
               valueFrom:
                 secretKeyRef:
                   name: "{{ .Values.backend.interfaces.secret }}"
-                  key: "clearinghouse-client-secret"
-            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__GRANTTYPE"
-              value: "{{ .Values.backend.processesworker.clearinghouse.grantType }}"
-            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__TOKENADDRESS"
-              value: "{{ .Values.clearinghouseTokenAddress }}"
-            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__PASSWORD"
+                  key: "default-clearinghouse-client-secret"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__DEFAULTCLEARINGHOUSECREDENTIALS__GRANTTYPE"
+              value: "{{ .Values.backend.processesworker.clearinghouse.defaultClearinghouse.grantType }}"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__DEFAULTCLEARINGHOUSECREDENTIALS__TOKENADDRESS"
+              value: "{{ .Values.defaultClearinghouse.tokenAddress }}"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__DEFAULTCLEARINGHOUSECREDENTIALS__PASSWORD"
               value: "{{ .Values.backend.placeholder }}"
-            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__SCOPE"
-              value: "{{ .Values.backend.processesworker.clearinghouse.scope }}"
-            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__USERNAME"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__DEFAULTCLEARINGHOUSECREDENTIALS__SCOPE"
+              value: "{{ .Values.backend.processesworker.clearinghouse.defaultClearinghouse.scope }}"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__DEFAULTCLEARINGHOUSECREDENTIALS__USERNAME"
               value: "{{ .Values.backend.placeholder }}"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__DEFAULTCLEARINGHOUSECREDENTIALS__COUNTRYALPHA2CODE"
+              value: "{{ .Values.backend.processesworker.clearinghouse.defaultClearinghouse.countryAlpha2Code }}"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__REGIONALCLEARINGHOUSECREDENTIALS__0__BASEADDRESS"
+              value: "{{ .Values.regionalClearinghouse.china.baseAddress }}{{ .Values.regionalClearinghouse.china.apiPath }}"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__REGIONALCLEARINGHOUSECREDENTIALS__0__CLIENTID"
+              value: "{{ .Values.backend.processesworker.clearinghouse.regionalClearinghouse.china.clientId }}"
+            - name: "APPLICATIONCHECKLIST__CLEARINGHOUSE__REGIONALCLEARINGHOUSECREDENTIALS__0__CLIENTSECRET"
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .Values.backend.interfaces.secret }}"
+                  key: "china-clearinghouse-client-secret"
```

I'd switch the key for a better overview to
```suggestion
                  key: "clearinghouse-client-secret-china"
```

---

done